### DISCUSSION TITLE
muduo能否支持epoll超时回调？？？

### DISCUSSION THREAD

**Discussion by VeblenXSYK** [chenshuo/muduo#586] [General]
陈老师，我这边需要在epoll超时的时候做些处理工作，请问一下现在的muduo支持epoll超时回调吗？？？如何不支持的话能够通过别的手段来达到同样的效果？？？

**Comment by andylau004** [comment]
Timestamp EPollPoller::poll(int timeoutMs, ChannelList* activeChannels)

第一个参数 timeoutMs不是超时么，

**Comment by chenshuo** [comment]
你直接用个 `Timer` 行不行？ `EventLoop::runEvery()`
